evennia/accounts/accounts.py

```diff
@@ -707,6 +711,40 @@ def set_password(self, password, **kwargs):
         logger.log_sec(f"Password successfully changed for {self}.")
         self.at_password_change()
 
+    def get_character_slots(self) -> typing.Optional[int]:
+        """
+        Returns the number of character slots this account has.
+
+        By default, that's settings.MAX_NR_CHARACTERS but this makes it easy to override.
+        Maybe for your game, players can be rewarded with more slots, somehow.
+
+        If it returns None, then there are no limits on character slots.
```

Comment:
I know you have added typing, but Evennia follows the google-style standard, which means you should nevertheless include a `Returns:` stanza in your doc strings.